Documentation/building/code-coverage.md

```diff
@@ -60,3 +60,18 @@ The results for this one library will then also show up in the aforementioned in
 And then once the run completes:
     
     ..\..\..\bin\tests\coverage\index.htm
+
+## Code coverage with mscorlib code
+
+Some code in the corefx contracts does not live in the corefx repo, e.g. System.Runtime, System.Text.Encoding. To run coverage reports for these projects, you need to build mscorlib locally from the coreclr repo.
+
+
+The following steps can be used manually to produce a coverage report, but a customizable batch file can be found [here](https://gist.github.com/hughbe/511331a3d264d48b390d1803475a91cf). Changing the parameters in the first couple of lines lets you run a coverage report easily for any mscorlib project.
```

Comment:
Thanks for putting the batch file together.  Rather than having it living in an arbitrary gist that folks need to copy out, should we actually check it into the repo somewhere, so folks can just run it?